sparkz-util/src/main/scala/sparkz/util/serialization/Reader.scala

```diff
@@ -0,0 +1,140 @@
+package sparkz.util.serialization
+
+
+trait Reader {
```

Comment:
good time to add `def getBoolean(): Boolean` method.
We had an issue before that we have `putBoolean` in the Writer, but miss the method in the Reader.
Update UTs as well